Cleanse/BindingBuilder.swift

```diff
@@ -70,37 +70,30 @@ extension Provider : _StandardProvider, _AnyStandardProvider {
 
 /// MARK: Building steps
 
-extension BindingBuilder where FinalProvider: _StandardProvider, Self.MaybeScope == _Unscoped {
+extension BindingBuilder where FinalProvider: _StandardProvider, Self.MaybeScope == Unscoped {
     /// Qualifies the provider being registered with a tag.
 
     public func tagged<Tag: Cleanse.Tag>(with tag: Tag.Type) -> TaggedBindingBuilderDecorator<Self, Tag> where Tag.Element == FinalProvider.Element {
         return self.with()
     }
 }
 
-extension BindingBuilder where MaybeScope == _Unscoped {
-    
-    public func asSingleton() -> ScopedBindingDecorator<Self, Singleton> {
-        return self.with()
+extension BindingBuilder where MaybeScope == Unscoped {
+    @available(*, unavailable, renamed: "sharedInScope()")
+    public func asSingleton() -> ScopedBindingDecorator<Self, Unscoped> {
+        preconditionFailure()
```

Comment:
it won't compile, so no error mesage required. Just needed something to make it not expect a return value